docs/howto/shell.rst

```diff
@@ -0,0 +1,27 @@
+Use the administration shell
+----------------------------
+
+The procrastinate shell is a tool to administrate jobs and overview queues and tasks.
+It is an interactive shell that you can run with the following command.
+
+*Experimental feature*.
+
+.. code-block:: console
+
+    $ procrastinate shell
+    Welcome to the procrastinate shell.   Type help or ? to list commands.
+
+    procrastinate> help
+
+    Documented commands (type help <topic>):
+    ========================================
+    EOF  cancel  exit  help  list_jobs  list_queues  list_tasks  retry
+
+As usual, you should use ``--app`` argument or ``PROCRASTINATE_APP`` environment
+variable to specify the application you want to use (see `command_line`).
+
+There are commands to list all the jobs (``list_jobs``), tasks (``list_tasks``)
+& queues (``list_queues``).
+And commands to retry (``retry``) & cancel (``cancel``) a specific job.
```

Comment:
Given that we have `list_jobs` & co. I've been wondering if having `retry_job` instead of `retry`, and `cancel_job` instead of `cancel`, would make things more consistent and explicit. When I open the shell for the first time and see `cancel` as a possible command I was not sure what that command would cancel. Curious what others think…

Reply:
I'm not sure if it's relevant because I see no confusion here.
What would be the meaning of "retry"/"cancel" applied to a task or a queue?

Reply:
Right. When I saw `cancel` for the first time I thought it would cancel the shell. But it may be just me :-) And `help cancel` immediately shed light on my confusion. Up to you really.